### ISSUE TITLE
Can't connect to WebIDE through browser

### ISSUE THREAD

**Issue by dyermaker504** [adafruit/Adafruit-WebIDE#221]
Using Occidentalis v0.2, and established wifi connections and ssh from PC to pi's local IP address (192.168.1.xx). On the pi, I can also browse with Midori and ping google.com. I followed the guide once and followed the prompts and suggestions, and then failed to get into the pi through the browser (it doesn't connect, firefox and chrome). So I uninstalled WebIDE through SSH and re-installed it and I'm getting the same output.

pi@raspberrypi ~ $ curl https://raw.github.com/adafruit/Adafruit-WebIDE/alpha/scripts/uninstall.sh | sudo sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1815  100  1815    0     0   5598      0 --:--:-- --:--:-- --:--:--  7260
***\* Removing restartd WebIDE configuration ****
sed: can't read /etc/restartd.conf: No such file or directory
sh: 15: restartd: not found
***\* Removing webide user from sudoers ****
/etc/sudoers.tmp: parsed OK
/etc/sudoers.d/README: parsed OK
***\* Removing access to port 80 for node ****
sh: 37: setcap: not found
***\* Stopping the Adafruit WebIDE ****
adafruit-webide.sh: unrecognized service
***\* Removing update-rc.d service ****
update-rc.d: using dependency based boot sequencing
rm: cannot remove `/etc/init.d/adafruit-webide.sh': No such file or directory
***\* Removing the WebIDE Folder ****
***\* Removing webide user ****
userdel: user 'webide' does not exist
***\* The Adafruit WebIDE is now uninstalled! ****
***\* During the installation process, there were a few ****
***\* libraries installed that we did not uninstall as ****
***\* we're not able to determine if other applications are dependent ****
***\* on them. If you are not using them, you can uninstall by executing ****
***\* the following command: ****
***\* sudo apt-get remove nodejs npm redis-server git restartd libcap2-bin avahi-daemon i2c-tools python-smbus ****
pi@raspberrypi ~ $ curl https://raw.github.com/adafruit/Adafruit-WebIDE/alpha/scripts/install.sh | sudo sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0false
100  4941  100  4941    0     0  14785      0 --:--:-- --:--:-- --:--:-- 19225
***\* Downloading the latest version of the WebIDE ****
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 22.6M  100 22.6M    0     0   527k      0  0:00:43  0:00:43 --:--:--  639k
***\* Installing required libraries ****
***\* (redis-server git restartd libcap2-bin avahi-daemon i2c-tools python-smbus) ****
Hit http://mirrordirector.raspbian.org wheezy InRelease
Hit http://archive.raspberrypi.org wheezy InRelease
Hit http://mirrordirector.raspbian.org wheezy/main armhf Packages
Hit http://archive.raspberrypi.org wheezy/main armhf Packages
Hit http://mirrordirector.raspbian.org wheezy/contrib armhf Packages
Hit http://mirrordirector.raspbian.org wheezy/non-free armhf Packages
Hit http://mirrordirector.raspbian.org wheezy/rpi armhf Packages
Ign http://archive.raspberrypi.org wheezy/main Translation-en_GB
Ign http://archive.raspberrypi.org wheezy/main Translation-en
Ign http://mirrordirector.raspbian.org wheezy/contrib Translation-en_GB
Ign http://mirrordirector.raspbian.org wheezy/contrib Translation-en
Ign http://mirrordirector.raspbian.org wheezy/main Translation-en_GB
Ign http://mirrordirector.raspbian.org wheezy/main Translation-en
Ign http://mirrordirector.raspbian.org wheezy/non-free Translation-en_GB
Ign http://mirrordirector.raspbian.org wheezy/non-free Translation-en
Ign http://mirrordirector.raspbian.org wheezy/rpi Translation-en_GB
Ign http://mirrordirector.raspbian.org wheezy/rpi Translation-en
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
i2c-tools is already the newest version.
python-smbus is already the newest version.
You might want to run 'apt-get -f install' to correct these:
The following packages have unmet dependencies:
 nodejs : Depends: libc-ares2 (>= 1.7.3) but it is not going to be installed
          Depends: libev4 (>= 1:4.04) but it is not going to be installed
          Depends: libv8-3.8.9.20 but it is not going to be installed
 python-dev : Depends: python2.7-dev (>= 2.7.3-1~) but 2.7.3~rc2-2.1 is to be installed
 python2.7-dev : Depends: python2.7 (= 2.7.3~rc2-2.1) but 2.7.3-6 is to be installed
                 Depends: libpython2.7 (= 2.7.3~rc2-2.1) but 2.7.3-6 is to be installed
 redis-server : Depends: libjemalloc1 (>= 2.1.1) but it is not going to be installed
E: Unmet dependencies. Try 'apt-get -f install' with no packages (or specify a solution).
pi@raspberrypi ~ $

At this point, I still can't connect through my browser on my PC.
So the next thing I did following the first install was "sudo apt-get -f install", but that didn't help. I've tried port 80 and 3000. Any ideas? Is there something I'm just not getting?

**Comment by fdominguezr** [comment]
Hi!

I can't access to the WebIDE that is in my Raspberry Pi B+, with my Google Chrome.
I tried with both http://raspberrypi.local and http://xx.xx.xx.xx where the x's are replaced with your Pi's IP address.
The message I get is: ERR_CONNECTION_REFUSED

Any idea? Thanks!

**Comment by joanne254** [comment]
@jwcooper Same issue here. I'm on Windows, Pi 2 Model B, install seemed to go okay-ish. I used the Easy Installation script. Had to comment out 2 sources that were breaking the update, and the service `restart` or `force-reload` always fail, but `start` and `stop` are fine and the service seems to be running.
I can ping my Pi, but can't manage to reach it via Chrome. I've tried `http://xxx.xxx.xxx.xxx` with the correct IP address, no chance :/ I get a "ERR_CONNECTION_REFUSED".
It'd be great to get some help with this!
